Pinta.Tools/Editable/EditEngines/ArrowedEditEngine.cs

```diff
@@ -243,106 +244,99 @@ protected override void DrawExtras (ref RectangleD? dirty, Context g, ShapeEngin
 	private Separator ArrowSeparator => arrow_sep ??= GtkExtensions.CreateToolBarSeparator ();
 	private Label ArrowLabel => arrow_label ??= Label.New (string.Format (" {0}: ", Translations.GetString ("Arrow")));
 
-	private CheckButton ArrowOneEnabledCheckBox {
-		get {
-			if (show_arrow_one_box is null) {
-				show_arrow_one_box = CheckButton.NewWithLabel ("1");
-				show_arrow_one_box.Active = settings.GetSetting (ARROW1_SETTING (tool_prefix), previous_settings_1.Show);
-				show_arrow_one_box.OnToggled += (o, e) => ArrowEnabledToggled (true);
-			}
+	private CheckButton ArrowOneEnabledCheckBox => show_arrow_one_box ??= Create_ArrowOneEnagledCheckBox ();
 
-			return show_arrow_one_box;
-		}
+	private CheckButton Create_ArrowOneEnagledCheckBox ()
```

Comment:
There is a typo here (`Enagled`) and I think the underscore is unnecessary

Reply:
No problem. I just fixed the spelling and removed the underscores from the names